### PR TITLE
rm galaxy from ecosystem

### PR DESCRIPTION
**Pull request by oraNod**
Drop old galaxy link. Relates to https://github.com/ansible/docsite/issues/194

### CODE REVIEW

**Review by Andersson007 — APPROVED** [review]

File: templates/ecosystem.html [hunk 1]
```diff
@@ -124,14 +124,6 @@
...
-            "name": "Galaxy",
-            "description": "Galaxy provides a hub for discovering and sharing Ansible community content.",
-            "docs": {
-                "url": "https://galaxy.ansible.com/docs/",
-                "label": "Visit Galaxy documentation"
-            }
-        },
         {
             "name": "Galaxy NG",
             "description": "Galaxy NG jumpstarts automation projects with Ansible community content.",
```
Comment (samccann) [inline templates/ecosystem.html:129]: FWIW - Galaxy just removed 'ng' from the website so maybe as part of this PR we can also just call it Galaxy here?

Comment (Andersson007) [reply templates/ecosystem.html:129]: FWIW yeah, Galaxy NG stands for new generation AFAIK which is a dangerous type of name:)

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Instead of removing, I would suggest replacing with current URL, which is https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/community/userguide/

File: templates/ecosystem.html (hunk 1 above)
Comment (oraNod) [reply templates/ecosystem.html:129]: I'm guessing from your comment in https://github.com/ansible/ecosystem-documentation/issues/23#issue-2038235310 that we continue to call this "Galaxy NG" - no change needed

**Pull request merged**